vm/chef/cookbooks/google-cloud-ops-agent/README.md

```diff
@@ -0,0 +1,143 @@
+# Google Cloud Operations Agents Chef Integration
+
+[![Linux Continuous Integration](https://github.com/GoogleCloudPlatform/google-cloud-ops-agents-chef/actions/workflows/linux.yml/badge.svg)](https://github.com/GoogleCloudPlatform/google-cloud-ops-agents-chef/actions/workflows/linux.yml)
```

Comment:
Instead of including the whole README, let's do a more simple one, referring the user to the official cookbook.

Reply:
Check docker container readmes first section.